cmd/storage.go

```diff
@@ -290,6 +299,22 @@ var blobberUpdateCmd = &cobra.Command{
 			updateBlobber.IsRestricted = &ia
 		}
 
+		var storageVersion int64
+		if flags.Changed("storage_version"){
+			if storageVersion, err = flags.GetInt64("storage_version"); err != nil{
+				log.Fatal(err)
+			}
+			updateBlobber.StorageVersion = &storageVersion
+		}
+
+		var managingWallet string
```

Comment:
Remove managing wallet, you can't update it.

Reply:
Removed